package/govuk-prototype-kit.config.json

```diff
@@ -1,14 +1,6 @@
 {
-  "nunjucksPaths": [
-    "/"
-  ],
-  "scripts": [
-    "/govuk/all.js"
-  ],
-  "assets": [
-    "/govuk/assets"
-  ],
-  "sass": [
-    "/govuk/all.scss"
-  ]
+  "nunjucksPaths": ["/"],
+  "scripts": ["/govuk/all.js"],
+  "assets": ["/govuk/assets", "/idsk/assets/"],
+  "sass": ["/govuk/all.scss"]
 }
```

Comment:
podla dokumentacie, package folder je generovany, nie som si isty ci to takto ostane aj po pregenerovani. Skusal si to ci to pojde? Pripadne sa spytat Feriho. 

https://github.com/id-sk/id-sk-frontend/blob/master/docs/contributing/application-architecture.md
package/ contains auto-generated files
package published on npm. Consume all of govuk-frontend through a single package.